tests/legacy-cli/e2e/tests/build/jit-prod.ts

```diff
@@ -0,0 +1,15 @@
+import { ng } from '../../utils/process';
+import { updateJsonFile } from '../../utils/project';
+
+
+export default async function () {
+  // Make prod use JIT.
+  await updateJsonFile('angular.json', configJson => {
+    const appArchitect = configJson.projects['test-project'].architect;
+    appArchitect.build.configurations['production'].aot = false;
+    appArchitect.build.configurations['production'].buildOptimizer = false;
+  });
+
+  // Test it works
+  await ng('e2e', '--prod');
```

Comment:
LGTM, though this is odd how it is working following yesterdays issue.

Reply:
Indeed.

For context, we saw a problem with Webpack `mode: 'production'` in unit tests in https://github.com/cexbrayat/r3-test-bed.